tslearn/metrics/utils.py

```diff
@@ -9,48 +9,39 @@ def _cdist_generic(dist_fun, dataset1, dataset2, n_jobs, verbose,
                    compute_diagonal=True, dtype=numpy.float, *args, **kwargs):
     """Compute cross-similarity matrix with joblib parallelization for a given
     similarity function.
-
```

Comment:
Could you please revert all these line suppressions? They tend to pack the docstrings.